catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/Status.java

```diff
@@ -38,6 +41,17 @@ public Status(QueryResponse response, String source, long elapsedTime) {
                 .size();
         hits = response.getHits();
         successful = isSuccessful(response.getProcessingDetails());
+        messages = getMessages(response.getProcessingDetails());
+    }
+
+    private ArrayList<String> getMessages(final Set<ProcessingDetails> details) {
+        ArrayList<String> messages = new ArrayList<String>();
+        for (ProcessingDetails detail : details) {
+            if (detail.hasException()) {
+                messages.add(detail.getException().getMessage());
+            }
+        }
+        return messages;
```

Comment:
Not necessary for approval, just optional improvement:
This method could also be succinctly written as
```java
return details.stream()
    .filter(ProcessingDetails::hasException) // only things with exceptions
    .map(detail -> detail.getException().getMessage()) // get the exception message
    .collect(Collectors.toList()) // collect into a list
```
and that map operation could also be
```java
    ...
    .map(ProcessingDetails::getException)
    .map(Exception::getMessage)
    ...
```